src/content/cloud/okteto-cli.mdx

```diff
@@ -76,6 +76,13 @@ In order to refer to images in the `build` section of the Okteto Manifest, you c
 - `${OKTETO_BUILD_<service>_REPOSITORY}`: the name of the image that was pushed (`namespace/_<service>`)
 - `${OKTETO_BUILD_<service>_IMAGE}`: the full image reference (using its SHA)
 
+It is also possible to handle timeouts from the client side when communicating with the buldkit daemon. To do so, the following environment variables can be modified:
```

Comment:
@rberrelleza @AdrianPedriza since this is an advanced scenario, I wouldn't include it in the main Okteto CLI doc. What do you think?

Reply:
I agree.

Reply:
Sounds good!

Reply:
Where do you want to document this @pchico83 ?

Reply:
Maybe an advanced configuration section in the okteto cli reference doc?
https://www.okteto.com/docs/reference/cli/

Reply:
Yeah, that's a better place! 

Or maybe in the doc for  `okteto build`? https://deploy-preview-164--okteto-docs.netlify.app/docs/reference/cli/#build

Reply:
Let's go with Pablo's suggestion if having an Advanced section in the CLI documentation for this information.